prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,47 @@
+const number1 = 10;
+const number2 = 20;
+
+const product = number1 * number2;
+
+console.log(product);
+console.log(typeof product);
+
+const charge = 10;
+const payment = 50;
+const amountRemaining = payment - charge;
+
+console.log(amountRemaining);
+console.log(typeof amountRemaining);
+
+const tests = 90;
+const assignments = 85;
+const final = 95;
+const grade = (tests + assignments + final) / 3;
+
+console.log(grade);
+console.log(typeof grade);
+
+const firstName = 'Coby';
+const lastName = 'Welty';
+const fullName = firstName + ' ' + lastName;
+
+console.log(fullName);
+console.log(typeof fullName);
+
+const pH = 12;
+const isAcidic = pH < 7;
+
+console.log(isAcidic);
+console.log(typeof isAcidic);
+
+const numberOfSoldiers = 300;
+const isSparta = numberOfSoldiers === 300;
+
+console.log(isSparta);
+console.log(typeof isSparta);
+
+const nickname = fullName;
+nickname += ' is the GOAT';
```

Comment:
This should actually stay as a `let`, because it gets reassigned on line 44

Reply:
whoops, added another commit to fix this